### PR TITLE
WFLY-19591 Community doc updates for MicroProfile OpenAPI 4.0.

### PR DESCRIPTION
**Pull request by pferraro**
https://issues.redhat.com/browse/WFLY-19591

### CODE REVIEW

File: docs/src/main/asciidoc/_admin-guide/subsystem-configuration/MicroProfile_OpenAPI.adoc [hunk 1]
```diff
@@ -77,13 +80,13 @@ e.g.
 $ curl -v http://localhost:8080/openapi?format=JSON
 < HTTP/1.1 200 OK
 ...
-{"openapi": "3.0.1" ... }
+{"openapi": "{OpenAPIVersion}.0" ... }
```
Comment (rhusar) [inline docs/src/main/asciidoc/_admin-guide/subsystem-configuration/MicroProfile_OpenAPI.adoc:83]: FWIW for some reason this doesn't render on GitHub - https://github.com/wildfly/wildfly/blob/44db5eefc1b340055f7b9e45cf23e947f99cf2df/docs/src/main/asciidoc/_admin-guide/subsystem-configuration/MicroProfile_OpenAPI.adoc#https-endpoint - but that doesn't matter.

**Review by rhusar — CHANGES_REQUESTED** [review]

**Comment by rhusar** [comment]
Also, no need to run CI on these. Cancelling manually.

File: docs/src/main/asciidoc/_admin-guide/subsystem-configuration/MicroProfile_OpenAPI.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (rhusar) [reply docs/src/main/asciidoc/_admin-guide/subsystem-configuration/MicroProfile_OpenAPI.adoc:83]: You need to add subs="attributes+" to the block:

```[source,shell,subs="attributes+"]```

File: docs/src/main/asciidoc/_admin-guide/subsystem-configuration/MicroProfile_OpenAPI.adoc [hunk 2]
```diff
@@ -77,13 +80,13 @@ e.g.
...
 
 
 $ curl -v -H'Accept: application/json' http://localhost:8080/openapi
 < HTTP/1.1 200 OK
 ...
-{"openapi": "3.0.1" ... }
+{"openapi": "{OpenAPIVersion}.0" ... }
```
[the remark this answers is not in the document]
Comment (pferraro) [reply docs/src/main/asciidoc/_admin-guide/subsystem-configuration/MicroProfile_OpenAPI.adoc:89]: This is the same code block.

File: docs/src/main/asciidoc/_admin-guide/subsystem-configuration/MicroProfile_OpenAPI.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (pferraro) [reply docs/src/main/asciidoc/_admin-guide/subsystem-configuration/MicroProfile_OpenAPI.adoc:83]: Perfect, thanks!

**Review by rhusar — APPROVED** [review]
Looks good now, ready to go @kabir

**Comment by rhusar** [comment]
Also, no need to run CI but cancelling it last time did not work.

**Pull request merged**